datamaxi/token/__init__.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict, Optional
+from datamaxi.api import API
+from datamaxi.lib.constants import BASE_URL
+
+
+class Token(API):
+    """Client to fetch Token status data from DataMaxi+ API."""
+
+    def __init__(self, api_key=None, **kwargs: Any):
+        """Initialize the object.
+
+        Args:
+            api_key (str): The DataMaxi+ API key
+            **kwargs: Keyword arguments used by `datamaxi.api.API`.
+        """
+        if "base_url" not in kwargs:
+            kwargs["base_url"] = BASE_URL
+        super().__init__(api_key, **kwargs)
+
+    def updates(
+        self,
+        type: Optional[str] = None,
+        page: int = 1,
+        limit: int = 1000,
+        sort: str = "desc",
+    ) -> Dict[str, Any]:
+        """Get Token Updates
+
+        `GET /api/v1/token/updates`
+
+        <https://docs.datamaxi.finance/api/datasets/token>
+
+        Args:
+            type (str): Update type
+            page (int): Page number
+            limit (int): Limit of data
+            sort (str): Sort order
+
+        Returns:
+            Token Updates data in list of dictionary
+        """
```

Comment:
@hayotbisonai We have to make check on type value.

```suggestion
        """
        if type not None and type not in ["listing", "delisting"]:
            raise ValueError("type must be either listing or delisting when set")
```